packages/ui/core-components/src/lib/atoms/inputs/slider/Slider.svelte

```diff
@@ -0,0 +1,100 @@
+<script context="module">
+	export const evidenceInclude = true;
+</script>
+
+<script>
+	import { INPUTS_CONTEXT_KEY } from '@evidence-dev/component-utilities/globalContexts';
+	import { getContext } from 'svelte';
+	const inputs = getContext(INPUTS_CONTEXT_KEY);
+	import SliderShadcn from '../../shadcn/slider/sliderShadcn.svelte';
+	import HiddenInPrint from '../shared/HiddenInPrint.svelte';
+
+	/////
+	// Component Things
+	/////
+
+	/** @type {string} */
+	export let title;
+
+	/** @type {string} */
+	export let name;
+
+	/** @type {number|undefined} */
+	export let min = 0;
+
+	if (min) {
+		// if min was user-supplied
+		min = Number(min);
+		if (isNaN(min)) {
+			// input must be a number
+			throw Error('min must be a number');
+		}
+	} else {
+		min = 0;
+	}
+
+	/** @type {number} */
+	export let max = 100;
+
+	if (max) {
+		// if min was user-supplied
+		max = Number(max);
+		if (isNaN(max)) {
+			// input must be a number
+			throw Error('max must be a number');
+		} else if (max < min) {
+			throw Error('max cannot be less than min');
+		}
+	} else {
+		max = 100;
+	}
+
+	/** @type {number} */
+	export let step;
+
+	/** @type {number} */
+	export let ticks = 1;
+
+	/** @type {boolean} */
+	export let showMaxMin = true;
+	$: showMaxMin = showMaxMin === 'true' || showMaxMin === true;
+
+	/** @type {boolean} */
+	export let hideDuringPrint = true;
+	$: hideDuringPrint = hideDuringPrint === 'true' || hideDuringPrint === true;
+
+	/** @type {number} */
+	export let defaultValue = 0;
+
+	/** @type {[number]} */
+	let value = [defaultValue];
+
+	$: $inputs[name] = value;
+
+	/** @type {string} */
+	export let size = '';
+
+	const renderSize = (size) => {
+		const sizeMap = {
+			medium: 'w-64',
+			large: 'w-96',
+			full: 'w-full'
+		};
+		return sizeMap[size.toLowerCase()] || 'w-40';
+	};
+
+	$: sizeClass = renderSize(size);
+</script>
+
+<HiddenInPrint enabled={hideDuringPrint}>
+	<div class={`relative ${sizeClass} mb-8`}>
+		<p class="pb-2 truncate text-xs">
+			{title} : <span class="text-sm">{$inputs[name]}</span>
+		</p>
+		<SliderShadcn {min} {max} {step} {ticks} bind:value />
```

Comment:
The slider position does not appear to currently be reactive to changes in the slider value

![CleanShot 2024-05-28 at 15 29 47](https://github.com/evidence-dev/evidence/assets/58074498/484e863b-d097-4b25-b2fd-7e86fcbdda32)

I think the linking of two sliders to the same input `monthsLarge` in the docs was probably unintentional, but it does illustrate a reactivity quirk

Reply:
interesting, resolving the linkage and taking a look at the reactivity quirk

Reply:
fixed the Large and Full slider connection

Was not able to experience the reactivity quirk on my side, would like to talk more about it when we can.

Thanks!

Reply:
I think you will need to keep the erroneous linkage in order to reproduce the issue.

this was on Firefox, for what it's worth